.github/workflows/ci.yml

```diff
@@ -56,11 +54,34 @@ jobs:
           save-if: false
 
       - name: Run all tests
-        run: cargo test --package ${{ matrix.crate }} --all-features
+        run: cargo test --all-features
```

Comment:
```suggestion
        run: cargo test --workspace --all-features
```

Reply:
this probably should be enough to test all the crates, and we can then just use the `gather_crates` for publishing.